Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs

```diff
@@ -278,16 +273,10 @@ private void TryLoadPointerIfNeeded()
                 // pointer currently registered with FocusManager, we use it.
 
                 Pointer = FocusManager.Instance.TryGetSinglePointer();
-
-                if (Pointer == null)
```

Comment:
Let me know what the verdict is on this, @NeerajW.  Ideally all our prefabs and test scenes have FocusManager setup correctly in them already (I know a couple months back, I made sure they all did in our project, but perhaps those changes didn't get propagated back into all tests scenes in this repo.  Let me know if you want me to check and fix them all.